build.gradle

```diff
@@ -72,7 +72,7 @@ ext.deps = [
 
 allprojects {
   group = "com.linkedin.tony"
-  project.version = "0.2.0"
```

Comment:
Why append `-SNAPSHOT`?

Reply:
I will revert that. This was for publishing to local maven repo to prevent conflicts with maven central's version.